.github/workflows/ci.yml

```diff
@@ -94,12 +96,40 @@ jobs:
           sudo update-alternatives --config gcc
           sudo apt-get install ninja-build fontconfig libfontconfig1-dev libglu1-mesa-dev libxrandr-dev libdbus-1-dev zip xvfb -y
           sudo Xvfb :0 -screen 0 1280x720x24 &
+      - run: |
+          # Test Linux native
           export DISPLAY=:0
-          ./gradlew --stacktrace --info -Pskiko.native.enabled=true -Pkotlin.native.cacheKind.linuxX64=none -Pskiko.test.onci=true :skiko:linuxX64Test :skiko:awtTest
-          ./gradlew --stacktrace --info -Pkotlin.native.cacheKind.linuxX64=none :skiko:publishToMavenLocal
-          ./gradlew --stacktrace --info :SkiaAwtSample:installDist # check jvm sample works
-          ./gradlew -Pskiko.android.enabled=true :skiko:publishSkikoJvmRuntimeAndroidX64PublicationToMavenLocal :skiko:publishSkikoJvmRuntimeAndroidArm64PublicationToMavenLocal :skiko:publishAndroidPublicationToMavenLocal
+          ./gradlew --no-daemon --stacktrace --info -Pskiko.native.enabled=true -Pkotlin.native.cacheKind.linuxX64=none -Pskiko.test.onci=true :skiko:linuxX64Test
+      - run: |
+          # Test Linux AWT
+          printJstackInBackground() {
+            # Temp function to print jstack stacktrace every 60 seconds
+            # Pay attention to org.jetbrains.skiko.redrawer.LinuxSoftwareRedrawer.dispose(LinuxSoftwareRedrawer.kt:27)
+            while [ 0 ]
+              do
+                echo "--------------------------------------------------------------------------"
+                echo "ps aux | grep java, result:"
+                ps aux | grep java
+                ps aux | grep java | grep jdk -i | awk '{print $2}' | while read pid; do
+                   jstack $pid
+                done
+                echo "--------------------------------------------------------------------------"
+                sleep 60
+              done
+          }
+          printJstackInBackground &
+          export DISPLAY=:0
+          ./gradlew --no-daemon --stacktrace --info -Pskiko.native.enabled=true -Pkotlin.native.cacheKind.linuxX64=none -Pskiko.test.onci=true :skiko:awtTest
+        timeout-minutes: 15
```

Comment:
15 minutes is not enough, even if tests run as expected. See the previous runs. They are near 15 minutes.

Reply:
It's timeout only for :skiko:awtTest.
In the previous try, it was just 5 minutes.

Reply:
15 is still small, even if it takes 5 minutes. Let's set 25